<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>29</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>생산지시서 종류:         </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radio_HY">
     <property name="text">
      <string>현대</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radio_OTIS">
     <property name="text">
      <string>OTIS</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radio_TKE">
     <property name="text">
      <string>TKE</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radio_ETC">
     <property name="text">
      <string>기타</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
